tests/test_data_entry_flow.py

```diff
@@ -285,6 +285,55 @@ async def async_step_finish(self, user_input=None):
     assert result["title"] == "Hello"
 
 
+async def test_show_progress(hass, manager):
+    """Test show progress logic."""
+    manager.hass = hass
+
+    @manager.mock_reg_handler("test")
+    class TestFlow(data_entry_flow.FlowHandler):
+        VERSION = 5
+        data = None
+
+        async def async_step_init(self, user_input=None):
```

Comment:
Could you extend this step that it will show two different progress actions before finishing progress. Ie, `show_progress("installing")` and `show_progress("configuring")` and then mark it as done.